src/components/Form/bookForm.css

```diff
@@ -1,32 +1,44 @@
 form {
-  display: grid;
-  place-items: center;
-  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  margin-top: 2rem;
+  margin-left: 3%;
+  margin-bottom: 3rem;
 }
 
-.author-1 {
-  border: 2px solid #000;
-  border-radius: 4px;
-  width: 85vw;
-  height: 3rem;
-  font-size: 20px;
-  margin-bottom: 1rem;
+.h2 {
+  font-size: 25px;
+  text-transform: uppercase;
+  color: #c5c5c5;
+  font-weight: 700;
 }
 
 .title-1 {
-  border: 2px solid #000;
+  background-color: transparent;
+  border: 1px solid #c5c5c5;
+  margin-right: 1rem;
+  width: 50vw;
+  height: 1.9rem;
+  padding-left: 1rem;
   border-radius: 4px;
-  width: 85vw;
-  height: 3rem;
-  font-size: 20px;
-  margin-bottom: 1rem;
 }
 
-.button {
-  border: 1px solid #fff;
+.author-1 {
+  background-color: transparent;
+  border: 1px solid #c5c5c5;
+  margin-right: 1rem;
+  width: 26vw;
+  height: 1.9rem;
+  padding-left: 1rem;
   border-radius: 4px;
-  font-size: 16px;
-  width: 85vw;
-  background-color: #000;
+}
+
+.button-5 {
+  font-size: 20px;
+  text-align: center;
   color: #fff;
+  font-weight: 100;
+  font-family: 'Courier New', Courier, monospace;
+  background-color: #0290ff;
+  border: none;
+  padding: 0.4rem 1.5rem 0.4rem 1.5rem;
+  border-radius: 4px;
 }
```

Comment:
- [ ] Please 🙏🏽 use the styles assigned to the add button in the original design to match with Zeplin design 👇🏽 

![image](https://github.com/Booze33/book-store/assets/58642949/a97b02c7-ae6e-4330-9534-2a15058b8ed8) ![image](https://github.com/Booze33/book-store/assets/58642949/43256d8f-56b3-4ae1-a1f3-3389a1c83dbd)